examples/demo-rollup/src/main.rs

```diff
@@ -24,9 +26,20 @@ pub fn initialize_ledger() -> LedgerDB {
     ledger_db
 }
 
+async fn rpc(module: RpcModule<()>, address: SocketAddr) {
```

Comment:
nit: should we rename this to `start_rpc_server` or something a little more descriptive?